### PR TITLE
Add Mash as a provider and templatize provider cards

### PR DESCRIPTION
**Pull request by jstol**
The various `styled` `img` components and inline `style` overriding was becoming a bit unwieldy, so this PR uses the same approach as in `community.js` to collect all providers into a list and apply it to a single `ProviderCard` template. It also adds [Mash](https://www.getmash.com/) as a provider (as we recently launched Lightning Addresses with our `@getmash.cash` domain ⚡).

(It also includes some minor cleanup, like removing an unused import in `hero.js`, and fixing some broken LNTXBot links throughout the site.)

### CODE REVIEW

File: components/providers.js [hunk 1]
```diff
@@ -239,6 +213,148 @@ const DomainURL = styled.div`
...
+    imageStyle: { width: '125px' },
+    lightningAddressDomain: 'bitrefill.me',
+    url: 'https://bitrefill.com/',
+  },
+  {
+    name: 'LNTXBot',
+    image: '/images/lntxbot.png',
+    imageStyle: { width: '130px', marginBottom: '-5px' },
+    lightningAddressDomain: 'lntxbot.com',
+    url: 'http://lntxbot.fiatjaf.com/',
```
Comment (jstol) [inline components/providers.js:244]: @fiatjaf just FYI, I noticed this URL is broken. Not sure if you'd like it swapped out for something else (like [this](https://fiatjaf.com/2495e7ef.html) or [this](https://telegram.me/lntxbot)).

Comment (fiatjaf) [reply components/providers.js:244]: Oh, please swap it for https://lntxbot.com if you can, please!

Comment (jstol) [reply components/providers.js:244]: will do ⚡

Comment (jstol) [reply components/providers.js]: made it uniform across the site so should be good now

**Comment by jstol** [comment]
@andrerfneves bumping in case this got lost – just wondering if you've had a chance to take a look at this PR yet. Thanks!

**Comment by andrerfneves** [comment]
Thank you for doing the lord's work. Ever since we went from 3 providers to over 15 I wanted to refactor this but haven't had time. Thank you!

**Comment by jstol** [comment]
@andrerfneves glad I could help! ⚡🙌🏻

**Pull request merged**